src/metakb/normalizers.py

```diff
@@ -83,188 +83,75 @@ def __init__(self, db_url: str | None = None) -> None:
     ) -> Allele | CopyNumberChange | CopyNumberCount | None:
-        """Normalize variation queries.
+        """Attempt to normalize a variation query
 
-        :param queries: Candidate query strings to attempt to normalize. Should be
-            provided in order of preference, as the result of the first one to normalize
-            successfully will be returned. Use in the event that a prioritized MANE
-            transcript is unavailable and multiple possible candidates are known.
+        :param query: Variation query to normalize
         :raises TokenRetrievalError: If AWS credentials are expired
         :return: A normalized variation, if available.
         """
-        for query in queries:
-            if not query:
-                continue
-            try:
-                variation_norm_resp = (
-                    await self.variation_normalizer.normalize_handler.normalize(query)
-                )
-                if variation_norm_resp and variation_norm_resp.variation:
-                    return variation_norm_resp.variation
-            except TokenRetrievalError as e:
-                _logger.error(e)
-                raise e
-            except Exception as e:
-                _logger.error(
-                    "Variation Normalizer raised an exception using query %s: %s",
-                    query,
-                    e,
-                )
+        try:
+            variation_norm_resp = (
+                await self.variation_normalizer.normalize_handler.normalize(query)
+            )
+            if variation_norm_resp and variation_norm_resp.variation:
+                return variation_norm_resp.variation
+        except TokenRetrievalError as e:
+            _logger.error(e)
+            raise e
+        except Exception as e:
+            _logger.error(
+                "Variation Normalizer raised an exception using query %s: %s",
+                query,
+                e,
+            )
         return None
 
-    def normalize_gene(
-        self, queries: list[str]
-    ) -> tuple[NormalizedGene | None, str | None]:
-        """Normalize gene queries.
-
-        Given a collection of terms, return the normalized concept with the highest
-        match (see the
-        `Gene Normalizer docs <https://gene-normalizer.readthedocs.io/latest/usage.html#match-types>`_ for
-        more details on match types, and how queries are resolved).
+    def normalize_gene(self, query: str) -> tuple[NormalizedGene, str | None]:
+        """Attempt to normalize a gene query
 
         >>> from metakb.normalizers import ViccNormalizers
         >>> v = ViccNormalizers()
-        >>> gene_terms = [
-        ...     "gibberish",  # won't match
-        ...     "NETS",  # alias
-        ...     "hgnc:1097",  # HGNC identifier for BRAF
-        ...     "MARCH3",  # previous symbol
-        ... ]
-        >>> v.normalize_gene(gene_terms)[0].normalized_id
+        >>> v.normalize_gene("BRAF")[0].normalized_id
```

Comment:
is the object that gets returned with normalize_gene() == what gets returned with _normalize_concept() (from the ViccNormalizers class definition). Why would you call both [0] and the .normalized_id here? 

Honestly I think my question is just that I am confused a little bit on what the ViccNormalizers() object looks like and how its intended to be used.

Reply:
Good call out -- normalized_id was removed (for gene, in [this](https://github.com/cancervariants/gene-normalization/pull/381) pr) so this is actually wrong. I'll go ahead and update